tests/pageObjects/LoginPage.js

```diff
@@ -0,0 +1,22 @@
+class LoginPage {
+    constructor(){
+        this.loginBtnLocator = ".btn-block";
```

Comment:
user term `selector`
```suggestion
        this.loginBtnSelector = ".btn-block";
```